packages/flutter/lib/src/widgets/table.dart

```diff
@@ -489,3 +507,30 @@ class TableCell extends ParentDataWidget<TableCellParentData> {
     properties.add(EnumProperty<TableCellVerticalAlignment>('verticalAlignment', verticalAlignment));
   }
 }
+
+@immutable
+class _TableSlot with Diagnosticable {
+  const _TableSlot(this.column, this.row);
+
+  final int column;
+  final int row;
+
+  @override
+  bool operator ==(Object other) {
+    if (other.runtimeType != runtimeType)
+      return false;
+    return other is _TableSlot
+        && column == other.column
```

Comment:
Indent nit.

Reply:
Indentation here is correct as per boilerplate in style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#common-boilerplates-for-operator-and-hashcode

Reply:
This is the first time I noticed this rule, thank you for your reminder!